save-demo/src/main/kotlin/com/saveourtool/save/demo/storage/ToolStorage.kt

```diff
@@ -14,9 +17,29 @@ private const val TOOLS_PATH = "tools"
  */
 @Component
 class ToolStorage(
-    private val configProperties: ConfigProperties,
+    configProperties: ConfigProperties,
 ) : AbstractFileBasedStorage<ToolKey>(Path.of(configProperties.fileStorage.location) / TOOLS_PATH) {
-    private val rootPath = Path.of(configProperties.fileStorage.location) / TOOLS_PATH
+    /**
+     * Todo: Implement DownloadToolService
+     */
+    init {
+        val supportedTools = listOf(
+            ToolKey("diktat", "1.2.3", "diktat"),
+            ToolKey("diktat", "1.2.3", "diktat.cmd"),
```

Comment:
why we need a cmd? for local run only?

Reply:
Yep, but it will be removed in next PR as far as we will download diktat.jar and ktlint